packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -906,6 +930,7 @@ export class ContainerRuntime extends TypedEventEmitter<IContainerRuntimeEvents>
             useDataStoreAliasing = false,
             flushMode = defaultFlushMode,
             enableOfflineLoad = false,
+            compressionOptions = {},
```

Comment:
I'd think it should be undefined, no? See my other comment about treatment of undefined

Reply:
As I said above, I would rather do it this way but we wrap the runtime options in `Readonly<Required<IContainerRuntimeOptions>>` in `ContainerRuntime`'s constructor so we're not able to pass in undefined here.